v19.2/gssapi_authentication.md

```diff
@@ -103,16 +132,43 @@ The `cockroach sql` shell does not yet support GSSAPI authentication. You need t
 {{site.data.alerts.end}}
 
 1. Install and configure your Kerberos client.
+
+In CentOS/RHEL - `yum install krb5-user`
```

Comment:
Is this specific for CentOS/RHEL? Do we have instructions for other systems?

Reply:
should be `apt-get install krb5-user`

Reply:
Done.